pkg/machine/wsl/machine.go

```diff
@@ -1013,6 +1014,10 @@ func (v *MachineVM) Stop(name string, _ machine.StopOptions) error {
 	return nil
 }
 
+func (v *MachineVM) State() (machine.Status, error) {
+	return "", define.ErrNotImplemented
+}
+
```

Comment:
I think this should be changed too.
In the qemu backend you changed isRunning() to State(). I think you should do the same for the wsl backend.
Right now inspect would not work on windows because of this.